### PR TITLE
39 camera

### PR DESCRIPTION
**Pull request by IssaHanou**
### Issue
Closes GH-39

## Contents
- creates a menu for the application, where user can navigate between three pages. 
- implements iframe for camera page

## TODO
- [x] read iframe src from config?

## Screens
big screen camera page:
image

small screen home page
image

### CODE REVIEW

File: back-end/resources/demo_config.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "name": "Demo week 6",
     "duration": "5m",
     "host": "192.168.178.82",
-    "port": 1883
+    "port": 1883,
+    "cameras": []
```
Comment (GwennanS) [inline back-end/resources/demo_config.json]: cameras are in this file on a different spot from the other configs

**Comment by GwennanS** [comment]
This also loads double cams when the backend is started after the front-end

**Comment by GwennanS** [comment]
I have a white bar below the main container on chrome

File: front-end/src/app/app.component.spec.ts [hunk 2]
```diff
@@ -56,13 +78,13 @@ describe("AppComponent", () => {
   it("should have as title 'S.C.I.L.E.R'", () => {
```
Comment (mrlomar) [inline front-end/src/app/app.component.spec.ts]: Change the description S.C.I.L.E.R > SCILER

**Comment by GwennanS** [comment]
everytime i switch screens the timer gains a tick

**Pull request merged**